### PR TITLE
(DOCSP-10364): Document mongosh logging

### PR DESCRIPTION
**Pull request by jwilliams-mongo**
[Jira](https://jira.mongodb.org/browse/DOCSP-10364)
[Stage](https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker/DOCSP-10364/logs)

### CODE REVIEW

File: source/reference/logs.txt [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+View |mdb-shell| Logs
+---------------------
+
+.. include:: /includes/steps/view-logs.rst
+
+View |mdb-shell| Command History
+--------------------------------
+
+|mdb-shell| saves a history of all commands you've run across sessions.
+When a new command is issued, it is added to the top of the history
```
Comment (jeff-allen-mongo) [inline source/reference/logs.txt:31]: [nit] Maybe "beginning" a better word than "top" here? "top" doesn't sound natural to me.

File: source/reference/logs.txt [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+file.
+
+.. tabs::
```
Comment (jeff-allen-mongo) [inline source/reference/logs.txt:34]: Do tabs add much here? I understand wanting to be consistent with the previous section, but I'm wondering if just a table with a row for macOS / Linux and a row for Windows might be cleaner. I think either approach could work, but just something to think about.

File: source/reference/logs.txt [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/admonitions/fact-mdb-shell-beta.rst
+
+|mdb-shell| stores logs for each session in `ndjson
+<http://ndjson.org/>`__ format using `pino
+<https://github.com/pinojs/pino>`__. With a Session ID, you can view or
```
Comment (jeff-allen-mongo) [inline source/reference/logs.txt:19]: [nit / wordsmith] "You can view or tail the logs for a |mdb-shell| session based on its session ID."

File: source/includes/steps-view-logs.yaml [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+        Connecting to: mongodb://127.0.0.1:27017
+        Using MongoDB: 4.2.8
+
+---
+title: "View the log for the session."
+level: 4
+ref: view-session-log
+content: |
+
+  .. tabs::
```
[the remark this answers is not in the document]
Comment (jwilliams-mongo) [reply source/includes/steps-view-logs.yaml:30]: updated

File: source/reference/logs.txt (hunk 3 above)
Comment (jwilliams-mongo) [reply source/reference/logs.txt]: updated

File: source/reference/logs.txt (hunk 1 above)
Comment (jwilliams-mongo) [reply source/reference/logs.txt]: good call. updated.

File: source/reference/logs.txt (hunk 2 above)
Comment (jwilliams-mongo) [reply source/reference/logs.txt]: updated. I think the table is cleaner.

**Comment by jwilliams-mongo** [comment]
Hi @mmarcon Can you please review this PR for logging in the new mongodb shell? Thank you!

**Comment by jeff-allen-mongo** [comment]
LGTM. Thanks @jwilliams-mongo

**Comment by mmarcon** [comment]
LGTM

@lrlna can you do a quick sanity check?

File: source/includes/steps-view-logs.yaml [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+        .. code-block:: sh
+
+           Get-Content %UserProfile%/.mongodb/mongosh/<sessionId>_log
+
+        Run the following PowerShell command to tail the log for a
+        session:
+
+        .. code-block:: sh
+
+           Get-Content %UserProfile%/.mongodb/mongosh/<sessionId>_log -Wait
```
Comment (lrlna) [inline source/includes/steps-view-logs.yaml:81]: You can also pipe a pretty-printer on windows! For example with `pino-colada` you'd do the exact same pipe as on Linux or MacOS:
```shell
Get-Content %UserProfile%/.mongodb/mongosh/<sessionId>_log -Wait | pino-colada
```
Annotation 2020-06-18 113317

File: source/includes/steps-view-logs.yaml [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+     .. code-block:: javascript
+        :copyable: false
+        :emphasize-lines: 3
+
+        $ mongosh
+
+        Current sessionID: c2961dbd6b73b052671d9df0
+        Connecting to: mongodb://127.0.0.1:27017
+        Using MongoDB: 4.2.8
```
Comment (lrlna) [inline source/includes/steps-view-logs.yaml:19]: We've changed the greeting message a bit with `0.0.6`. I am not sure how accurate you'd want to be, but this is what it looks like now:
```shell
Current sessionID:  c6d052d327b75c50c285d30f
Connecting to:      mongodb://127.0.0.1:27017
Using MongoDB:      4.2.5
Using Mongosh Beta: 0.0.6
```

Comment (jwilliams-mongo) [reply source/includes/steps-view-logs.yaml:19]: updated

File: source/includes/steps-view-logs.yaml (hunk 5 above)
Comment (jwilliams-mongo) [reply source/includes/steps-view-logs.yaml:82]: updated

**Comment by jwilliams-mongo** [comment]
thanks for your feedback, @lrlna!

**Pull request merged**